src/components/Pagination.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import styled from 'styled-components';
+import { palette } from "../theme/palette";
+
+export const LinkForPage = styled(({ page, current, href, onClick, className }: {
+  page: number;
+  current?: boolean;
+  href?: string;
+  className?: string;
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void;
+}) => {
+  const currentValue = current ? "page" : undefined;
+
+  return (
+    <a
+      className={className}
+      aria-current={currentValue}
+      href={href || '#'}
```

Comment:
Maybe we won't ever see this in practice, but with "#" the screenreader reads all of them as "visited link".

Reply:
Yea, this would be invalid maybe I should just make href required